src/utils/rss/generate-feed.js

```diff
@@ -46,7 +46,13 @@ const generateItem = function generateItem(post) {
 
 const generateRSSFeed = function generateRSSFeed(siteConfig) {
     return {
-        serialize: ({ query: { allGhostPost } }) => allGhostPost.edges.map(edge => Object.assign({}, generateItem(edge.node))),
+        serialize: ({ query: { allGhostPost } }) => {
+            if (allGhostPost) {
```

Comment:
Can you try 
`return allGhostPost && allGhostPost.edges.map(edge => Object.assign({}, generateItem(edge.node))) || []`

Reply:
Changed